include/eve/arch/arm/sve/sve_true.hpp

```diff
@@ -24,4 +24,24 @@ sve_true()
   else if constexpr( sizeof(T) == 4 ) return svptrue_b32();
   else if constexpr( sizeof(T) == 8 ) return svptrue_b64();
 }
-} // namespace eve::detail
+
+// Returns clear sve_true for type of a given cardinal
+// while masking potential garbage value
+template<relative_conditional_expr C, typename T>
+EVE_FORCEINLINE svbool_t
+sve_true(C cond, as<T> tgt)
+{
+  if constexpr(C::is_complete && C::is_inverted)
+  {
+    using v_t   = element_type_t<T>;
+    using ec_t  = expected_cardinal_t<v_t, typename T::abi_type>;
+
+    if constexpr(ec_t::value == T::size())      return sve_true<v_t>();
```

Comment:
Hmm. I think this should actually be: `T::size() < native_cardinal_v<v_t>`

Reply:
no, if it is smaller, then we got garbage. Remeber than SVE size are one at a time, so haiving SVE512 has no SVE256.